config/domain-mapping/300-resources/domain-claim.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: domainclaims.serving.knative.dev
```

Comment:
This feels like a networking internal thing to me 🤔 

Generally stuff in `serving.knative.dev` is public API surface, so I'd definitely avoid it here.

cc @tcnghia @dprotaso for thoughts

Reply:
Yeah, ack that externally visible might be wrong (although I think we *do* plan to let operators interact with it, at least in the multitenant case, which does argue more for visible/serving..). Also not totally sure networking is right since it's a bit higher level than ingress etc 🤔.

Reply:
Ack the operator element.  Perhaps `networking.knative.dev` (no internal) is a middle ground...?

cc @knative/networking-wg-leads  for thoughts.

Reply:
`networking` sounds good to me. `networking` does not just imply ingress. We also use `networking` for serverlessservice CRD https://github.com/knative/serving/blob/master/config/core/300-resources/serverlessservice.yaml#L18

Reply:
Serverlessservice is a good point. Ok, unless someone shouts I'll rework this to be in networking tomorrow. It's alpha so we can always move it back to serving if we change our mind later anyway.